# Josh L. Espinoza, Ph.D
<img src="profile.jpg" alt="isolated" width="300" class="center"/>

### About Me:
I am a multidisciplinary computational research scientist investigating topics in microbial ecology, systems biology, and antibiotic discovery. My expertise is in metagenomics, (un)supervised machine learning, network analysis, next-generation sequencing pipeline development, and feature engineering/selection. I also dabble in eukaryotic gene modeling (plants, diatoms, corals), compositional data analysis, epigenetics, and (Python) package development.

My research path prior to my doctoral studies has meandered through several domains including regenerative medicine, synthetic biology, and microbial ecology with an emphasis in machine-learning, network analysis, and predictive modeling.  As an undergraduate researcher at SDSU, I explored many different scientific opportunities to find a field that called to me.  More specifically, I performed research in ornithology, microbial ecology, and regenerative medicine laboraties while developing the basis for my computational skillset over the years.  During my senior year at SDSU, I was accepted into an exclusive joint internship program between CIRM, UCSD, and SDSU studying mammalian regenerative medicine providing me with an opportunity to showcase my informatics abilities in a practical setting.  This program naturally led to my graduate research at UCSC and an internship at JCVI where I formally developed my informatics expertise and statistics background to leverage machine learning towards exploring questions in biotechnology.  During my joint doctorate at JCVI and DUT, I developed domain-agnostic machine learning and network analysis methodologies that can be leverage towards a diverse array of scientific questions.  

My experience has allowed me to research a wide range of topics in prokaryotic, eukaryotic, and mixed systems from both biological, ecological, and computational perspectives.  As my research scope is wide, I have prioritized generalization, reusability, and flexibility in my computational methods, algorithms, and bioinformatics/visualization software packages. My approach to science is interdisciplinary and exploratory by building on existing methods, restructuring data in novel ways, and applying methods from other disciplines to solve biological questions. 

I consider myself an environmentalist and while many of my projects investigate human health and disease, the methods I develop are intended to be broadly applicable to other research questions.  In particular, I am interested in identifying microbes that can be used in biotechnology (e.g., plastic degrading enzymes, drug discovery), biodiversity, how interactions between microbes change in relation to a warming climate, ecological preservation, and [planetary intelligence](https://www.cambridge.org/core/journals/international-journal-of-astrobiology/article/intelligence-as-a-planetary-scale-process/5077C784D7FAC55F96072F7A7772C5E5).

Aside from research, I [draw things](art/README.md) and [produce music](https://soundcloud.com/o-rka/tracks) from time-to-time; though, my devotion to research has slowed this down.  I'm in coastal Southern California so rock climbing and diving are how I get in it as I seek inspiration from nature.  My core philosophy is that there's only one Earth (that is accessible) and humans are (arguably) the only known instance of advanced intelligence (i.e., influence over the geosphere > biosphere > technosphere).  It's our responsibility as humans to preserve the natural ecosystems that  have provided us with the opportunity to develop intelligence while progressing our unique instance of intelligence sustainably with nature.

### Affiliations:
* *J. Craig Venter Institute*

### Education:
* Ph.D in Biotechnology, *Durban University of Technology* (2022)
* M.Sc in Bioinformatics, *University of California Santa Cruz* (2016)
* B.Sc in Cellular and Molecular Biology, *San Diego State University* (2013)

### Research Highlights:
* Developed the [*VEBA*](https://github.com/jolespin/veba) modular end-to-end suite for in silico recovery, clustering, and analysis of prokaryotic, microeukaryotic, and viral genomes from metagenomes 
	* [doi:10.1186/s12859-022-04973-8](doi.org/10.1186/s12859-022-04973-8)

* Developed an explainable artificial intelligence methodology including novel feature selection algorithm (*Clairvoyance*), a Hierarchical Ensemble of Classifiers framework (HEC models), and a data analysis Python package (*Soothsayer*). These methods were developed to address challenges in antibiotic discovery pipeline but were designed with broad applicability. Collaboratively identified and characterized Darobactin (an antibiotic with novel activity isolated from the gut microbiome of nematodes) using said methodologies. 
	* [doi:10.1371/journal.pcbi.1008857](https://doi.org/10.1371/journal.pcbi.1008857)
	* [doi:10.1038/s41586-019-1791-1](https://www.nature.com/articles/s41586-019-1791-1)

* Developed a compositional feature engineering approach and an ensemble network analysis methodology that was extended for differential networks and sample-specific perturbation networks. These methods were developed to investigate dental caries and acute malnutrition but were also designed with broad applicability.
	* [doi:10.1093/pnasnexus/pgac239](https://doi.org/10.1093/pnasnexus/pgac239)
	* [doi:10.1016/j.ebiom.2021.103644](https://doi.org/10.1016/j.ebiom.2021.103644)
* Demonstrated how feature selection algorithm can be leveraged for identifying inferred interactions that are predictive of phenotypes.
 	* [doi:10.1016/j.ebiom.2021.103644](https://doi.org/10.1016/j.ebiom.2021.103644) 
* Collaboratively identified beneficial microbes that are help corals withstand rising ocean temperatures.
	* [doi:10.1126/sciadv.abg3088](https://www.science.org/doi/10.1126/sciadv.abg3088)


### Publications: 
* [https://github.com/jolespin/publications](https://github.com/jolespin/publications)

### Contact:
* jespinoz[A|T]jcvi[dot]org
* [https://www.linkedin.com/in/jolespin](https://www.linkedin.com/in/jolespin)